src/pe/section_table.rs

```diff
@@ -171,7 +208,7 @@ impl ctx::SizeWith<scroll::Endian> for SectionTable {
     }
 }
 
-impl ctx::TryIntoCtx<scroll::Endian> for SectionTable {
+impl ctx::TryIntoCtx<scroll::Endian> for &SectionTable {
```

Comment:
interesting, i wasn't aware you could do this; one longstanding issue in scroll (and major 1.0 blocker) was to allow pwrite impls for reference types, but it hadn't occurred to me the user could just write the TryIntoCtx impl in this manner, i'm not sure why it never was brought up in that issue...

Reply:
Yeah, I realized that there's almost no reason to give owned types to the serialization process given that you are really just reading those memory locations and discarding them, having _always_ a reference makes them cheap to move around.